build_msvc/vcpkg.json

```diff
@@ -13,5 +13,12 @@
       "features": ["thread"]
     },
     "zeromq"
+  ],
+  "builtin-baseline": "b86c0c35b88e2bf3557ff49dc831689c2f085090",
+  "overrides": [
+    {
+      "name": "libevent",
+      "version": "2.1.12+20230128"
```

Comment:
I think that is a fair point. The issue is that if we ever need to upgrade libevent due to security vulnerability it will break and you'll have to figure this out again and change `build_msvc/bitcoin_config.h.in`. This addresses that issue today rather then at some point in the future. That said, if we expect to move to CMake soon, then it makes sense to move to the previous version and get the minimum diff.

I'll change this to `2.1.12#7`, but I'm open to changing to back to `2.1.12+20230128`.